expr/func_summarize.go

```diff
@@ -87,7 +86,7 @@ func (s *FuncSummarize) Exec(cache map[Req][]models.Series) ([]models.Series, er
 func summarizeValues(serie models.Series, aggFunc batch.AggFunc, interval, start, end uint32) []schema.Point {
 	out := pointSlicePool.Get().([]schema.Point)
 
-	numPoints := int(util.Min(uint32(len(serie.Datapoints)), (start-end)/interval))
+	numPoints := len(serie.Datapoints)
```

Comment:
So it's okay to lose the check with start/end? 

The alternative is to default to len(serie.Datapoints) if it's over the max for uint32.

Reply:
I'm not sure what that check was trying to do. start-end will always be negative, which will cause an underflow for uint32.

Reply:
I was going based off of the python implementation: https://github.com/graphite-project/graphite-web/blob/master/webapp/graphite/render/functions.py#L4794

This seems to indicate that the other check should have been on the original start/end (not newStart / newEnd) and adjusted by the `step` (not the new `interval`). In MetricTank, this should already be null filled, so I'm not sure we need an additional check.